examples/dt/uni04delta/control-plane/service-values-wo-ceph-client.yaml

```diff
@@ -0,0 +1,144 @@
+      rbd_store_user = openstack
+    glanceAPIs:
+      default:
+        replicas: 3
+
+  manila:
+    enabled: true
+    manilaShares:
+      share1:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_share_backends = cephfs
+          enabled_share_protocols = cephfs
+
+          [cephfs]
+          driver_handles_share_servers = False
+          share_backend_name = cephfs
+          share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
+          cephfs_conf_path = /etc/ceph/ceph.conf
+          cephfs_cluster_name = ceph
+          cephfs_auth_id = openstack
+          cephfs_volume_mode = 0755
+          cephfs_protocol_helper_type = CEPHFS
+    manilaAPI:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_share_protocols=nfs,cephfs
+
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      dns_domain = openstackgate.local
+      vlan_transparent = true
+      service_plugins = qos,ovn-router,trunk,segments,port_forwarding,log
+      dhcp_agent_notification = false
+      agent_down_time = 600
+      router_distributed = true
+      router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+      allow_automatic_l3agent_failover = true
+      debug = true
+      log_dir = /var/log/neutron
+      [agent]
+      report_interval = 300
+      [database]
+      max_retries = -1
+      db_max_retries = -1
+      [keystone_authtoken]
+      region_name = regionOne
+      memcache_use_advanced_pool = True
+      auth_type = password
+      [nova]
+      region_name = regionOne
+      endpoint_type = internal
+      [oslo_messaging_notifications]
+      driver = noop
+      [oslo_middleware]
+      enable_proxy_headers_parsing = true
+      [oslo_policy]
+      policy_file = /etc/neutron/policy.yaml
+      [placement]
+      region_name = regionOne
+      [ovs]
+      igmp_snooping_enable = true
+      [ovn]
+      ovn_emit_need_to_frag = true
+      [ml2]
+      type_drivers = geneve,vlan
+      tenant_network_types = geneve,vlan
+      extension_drivers = qos,port_security,dns_domain_ports
+      [ml2_type_geneve]
+      vni_ranges = 1:65536
+      max_header_size = 38
+      [ml2_type_vlan]
+      network_vlan_ranges = tenant:1000:2000,datacentre:218:218
+
+  nova:
+    customServiceConfig: |
+      [filter_scheduler]
+      enabled_filters = AggregateInstanceExtraSpecsFilter, AvailabilityZoneFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,DifferentHostFilter,SameHostFilter,AllHostsFilter,ServerGroupAffinityFilter,ServerGroupAntiAffinityFilter
+
+  extraMounts:
+    - name: v1
+      region: r1
+      extraVol:
+        - propagation:
+            - CinderVolume
+            - CinderBackup
+            - GlanceAPI
+            - ManilaShare
+          extraVolType: Ceph
+          volumes:
+            - name: ceph
+              projected:
+                sources:
+                  - secret:
+                    name: ceph-conf-files
```

Comment:
indentation needs to be fixed here
secret:
  name: ceph-conf-files

Reply:
Fixed